packages/@o3r/apis-manager/README.md

```diff
@@ -112,13 +127,10 @@ class MyClass {
 }
 ```
 
-> [!WARNING]
-> Do not forget to import the **ApiManagerModule** in you component module
-
 ### Enforce custom API usage
 
 Some users may want to enforce existing components or services to use a specific Sdk instead of default API SDK.
-To do so the **INITIAL_APIS_TOKEN** will allow to indicate to the **ApiFactory** the class they will need to use (instead of default ones).
+To do so, the **INITIAL_APIS_TOKEN** will allow to indicate to the **ApiFactory**, the class they will need to use (instead of default ones).
```

Comment:
```suggestion
To do so, the **INITIAL_APIS_TOKEN** will allow to indicate to the **ApiFactory** which class it must use (instead of default ones).
```